test/models/organization_test.rb

```diff
@@ -1,6 +1,64 @@
 require 'test_helper'
 require 'models/shared/taxonomies_base_test'
+require 'rfauxfactory'
 
 class OrganizationTest < ActiveSupport::TestCase
   include TaxonomiesBaseTest
+
+  # List of valid organization names.
+  def valid_org_name_list
+    # Note: The maximum allowed length of org name is 242 only. This is an
+    # intended behavior (Also note that 255 is the standard across other
+    # entities.)
+    [
+      RFauxFactory.gen_alpha(1),
+      RFauxFactory.gen_alpha(242),
+      *RFauxFactory.gen_strings(1..242, exclude: [:html]).values,
+      RFauxFactory.gen_html(rand((1..217)))
+    ]
+  end
+
+  # List of invalid organization names.
+  def invalid_org_name_list
+    [
+      '',
+      ' ',
+      "\t",
+      *RFauxFactory.gen_strings(243).values
+    ]
+  end
+
+  test "create with multi names" do
+    valid_org_name_list.each do |name|
+      organization = FactoryBot.create(:organization, :name => name)
+      organization.reload
+      assert_equal organization.name, name
+    end
+  end
+
+  test "should not create with invalid names" do
+    invalid_org_name_list.each do |name|
+      organization = Organization.new(:name => name)
+      refute organization.valid?
+      assert_includes organization.errors.keys, :name
+    end
+  end
+
+  test "create with multi name and description" do
+    valid_org_name_list.each do |name|
+      organization = FactoryBot.create(:organization, :name => name, :description => name)
```

Comment:
I think this test should not go over all valid names, this is already tested by `create with multi names` test, the only added value is that it verifies description can be also saved. I think this is a crucial funcitonality of rails framework and does not have to be explicitly tested. If you want to have such test then I'd recommend keeping just one creation that uses description. But since I've seen you also added test for controller that uses this same path, I'd say this test should be dropped.